TestRailAutomationTest/Test/LoginPageTest.cs

```diff
@@ -18,11 +18,15 @@ public void Init()
     }
     
     [Test]
-    public void Test()
+    public void LoginTest()
     {
-        var homePageUrl = DataReader.GetConfig().HomePageUrl;
-        driver.Url = homePageUrl;
-        Assert.True(driver.Url.Contains(homePageUrl));
+        var user = UserCreator.CreateUser();
+        new LoginPage(driver)
```

Comment:
Are we sure that login page was opened?

Reply:
![image](https://user-images.githubusercontent.com/59253494/152765706-44e80bdf-d7fe-4530-9177-55dc46d253ed.png)